KEEP/slocate.bigfile.patch

```diff
@@ -0,0 +1,40 @@
+--- src/utils.c.orig	2006-01-06 00:16:19.000000000 +0000
++++ src/utils.c	2024-02-23 04:40:48.124456852 +0000
+@@ -13,6 +13,8 @@
+  *    GNU General Public License for more details.
+  *****************************************************************************/
+ 
++#define _LARGEFILE64_SOURCE
++
+ #include <stdio.h>
+ #include <stdarg.h>
+ #include <stdlib.h>
+@@ -284,12 +286,12 @@
+ verify_slocate_db(struct g_data_s *g_data, char *file)
+ {
+ 	char ch[1];
+-	struct stat tf_stat;
++	struct stat64 tf_stat;
```

Comment:
this patch is utterly wrong - the *64 functions and types are non-standard and on glibc are meant to be used _behind the scenes_ if you `#define _FILE_OFFSET_BITS 64`. that's what the original author of the patch should have done instead of hardcoding these hacks. on musl, everything off_t related is already 64bit by default.
btw, the plan in musl is to remove the *64 stuff completely in the next release, so this would have worked only till the next musl update.

Reply:
solved (bigfile patch removed)